### DISCUSSION TITLE
一种简单的 Hugo 本地搜索实现

### DISCUSSION THREAD

**Discussion by giscus** [loveminimal/comment#29] [General]
# 一种简单的 Hugo 本地搜索实现

一种简单的 Hugo 本地搜索实现 - https://aituyaa.com/%E4%B8%80%E7%A7%8D%E7%AE%80%E5%8D%95%E7%9A%84-hugo-%E6%9C%AC%E5%9C%B0%E6%90%9C%E7%B4%A2%E5%AE%9E%E7%8E%B0/

https://aituyaa.com/%E4%B8%80%E7%A7%8D%E7%AE%80%E5%8D%95%E7%9A%84-hugo-%E6%9C%AC%E5%9C%B0%E6%90%9C%E7%B4%A2%E5%AE%9E%E7%8E%B0/

**Comment by rebron1900** [comment]
大佬，关于你那篇解析wikilink的文章，我测试了一下发现在pre标签里的[[]]也会被解析，请问该如何过滤呢？（不好意思，相同的问题我还在你的github主题仓库里提了。

**Reply by loveminimal** [reply]
关于 wikilink 这个是对正文序列化之后，使用正则全局替换的，所以目前是没有办法区分是不是 pre 标签里面的内容~

**Comment by rebron1900** [comment]
@loveminimal 好的，只能换方案了。

**Reply by loveminimal** [reply]
是的，正则替换比较简单粗暴~
